ui/components/src/RefreshIntervalPicker/RefreshIntervalPicker.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, FormControl, MenuItem, Select, Stack } from '@mui/material';
+import { DurationString, RelativeTimeRange } from '@perses-dev/core';
+import { TimeOption } from '../model';
+
+interface RefreshIntervalPickerProps {
+  timeOptions: TimeOption[];
+  value?: RelativeTimeRange;
+  onChange: (value: RelativeTimeRange) => void;
+  height?: string;
+}
+
+export function RefreshIntervalPicker(props: RefreshIntervalPickerProps) {
+  const { value, onChange, timeOptions, height } = props;
+  const formattedValue = value?.pastDuration;
+  return (
+    <Stack direction="row" spacing={1} height={height}>
```

Comment:
i think it's ok to remove the `Stack` here since it only has one child component here

Reply:
Yup, makes sense.